src/main/java/org/apache/pirk/encryption/PrimeGenerator.java

```diff
@@ -18,15 +18,15 @@
  */
 package org.apache.pirk.encryption;
 
-import org.apache.pirk.utils.SystemConfiguration;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.HashMap;
 import java.util.Random;
 
+import org.apache.pirk.utils.SystemConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
I wonder if we are getting into an IDE commit-fest here, where one IDE thinks the imports should be ordered one way, and another brand of IDE thinks differently.